UT4MasterServer.Web/src/enums/account-flag.ts

```diff
@@ -0,0 +1,8 @@
+export enum AccountFlag {
+    None = 'None',
+    Admin = 'Admin',
+    Moderator = 'Moderator',
+    Developer = 'Developer',
+    ContentCreator = 'ContentCreator',
+    HubOwner = 'HubOwner'
+}

```

Comment:
Add numbers in comments from `UT4MasterServer.Models.AccountFlags` enum

Reply:
The enum numbers aren't relevant to the front end